raiden/tests/utils/transport.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+import logging
+
+import gevent
+
+from raiden.messages import decode
+from raiden.network.transport import (
+    DummyPolicy,
+    DummyTransport,
+)
+from raiden.utils import pex, sha3
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class MessageLoggerTransport(DummyTransport):
+    def __init__(
+            self,
+            host,
+            port,
+            protocol=None,
+            throttle_policy=DummyPolicy()):
+
+        super(MessageLoggerTransport, self).__init__(host, port, protocol, throttle_policy)
+        self.addresses_to_messages = {}
```

Comment:
can you initialize the dict with `dict()`? As I said above I thought we had agreed on this. It's not in the style guide though. If we are still in agreement about it I will add it.